sound/soc/intel/boards/sof_sdw_rt_sdca_jack_common.c

```diff
@@ -17,11 +17,13 @@
 #include <sound/jack.h>
 #include "sof_sdw_common.h"
 
+static int part_id;
```

Comment:
do we really want a static variable? IIRC it's possible to have solutions with multiple jacks for e.g. front and back

Reply:
> do we really want a static variable? IIRC it's possible to have solutions with multiple jacks for e.g. front and back

@plbossart Thanks for the reminding. Using component->name_prefix to determine codecs now.